subsys/nfc/ndef/le_oob_rec.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <string.h>
+#include <errno.h>
+#include <nfc/ndef/le_oob_rec.h>
+
+#define AD_TYPE_FIELD_SIZE 1UL
+#define AD_LEN_FIELD_SIZE 1UL
+
+#define LE_ROLE_PAYLOAD_SIZE 1UL
+
+/* Record Payload Type for Bluetooth Carrier Configuration LE record */
+const u8_t nfc_ndef_le_oob_rec_type_field[] = {
+	'a', 'p', 'p', 'l', 'i', 'c', 'a', 't', 'i', 'o', 'n', '/', 'v', 'n',
+	'd', '.', 'b', 'l', 'u', 'e', 't', 'o', 'o', 't', 'h', '.', 'l', 'e',
+	'.', 'o', 'o', 'b'
+};
+
+static int ad_encode(struct bt_data *ad, u8_t **buff, size_t *size)
```

Comment:
bt_data is more generic than adv data, so I think it would be helpful to rename to bt_data_encode

Reply:
fixed